src/ModelClient.ts

```diff
@@ -24,6 +24,13 @@ export class ModelClient {
         this._apiHost = apiHost || '';
         this._fetchPromises = {};
     }
+    /**
+     * Returns http host of the API
+     * @returns {string|null}
+     */
+    get apiHost(): string {
+        return this._apiHost == null ? '' : this._apiHost;
```

Comment:
`this._apiHost = apiHost || null;`

we have to be very specific here, `''` is a subset of string which will be rather confusing, so either `null` or string imo